MediaFeeder/MediaFeeder/PlaybackManager/PlaybackSession.cs

```diff
@@ -36,6 +41,29 @@ public void Dispose()
         _manager.RemoveSession(this);
     }
 
+    public List<Video> Playlist => _playlist;
```

Comment:
Given the use case, would a queue make more sense than a list for this?
Also, is there a particular reason for having a getter and a backing field rather than a property?